### PR TITLE
Initial Statistics Screen

### PR DESCRIPTION
**Pull request by mail2bishnoi**
Hi Anton,

Implemented Statistics Screen. Please review and suggest improvements.

### CODE REVIEW

File: modern/src/reports/ReportFilter.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import t from '../common/localization';
 import { useSelector } from 'react-redux';
 import moment from 'moment';
 
-const ReportFilter = ({ children, handleSubmit, showOnly }) => {
+const ReportFilter = ({ children, handleSubmit, showOnly, showDevices }) => {
```
Comment (tananaev) [inline modern/src/reports/ReportFilter.js:7]: If you want to re-use the filter, I would recommend adding more nesting. I don't like adding another configuration here.

For example, you can have `ReportFilter` without device and `DeviceReportFilter` with device that uses `ReportFilter`.

**Review by tananaev — COMMENTED** [review]
Please also share screenshots.

File: modern/src/reports/ReportFilter.js (hunk 1 above)
Comment (mail2bishnoi) [reply modern/src/reports/ReportFilter.js:7]: I would not like to complicate things for now. Just separated the statistics page and removed the configuration from ReportFilter.

image

Comment (tananaev) [reply modern/src/reports/ReportFilter.js:7]: Can we revert these changes then?

Comment (mail2bishnoi) [reply modern/src/reports/ReportFilter.js:7]: Done.

Comment (tananaev) [reply modern/src/reports/ReportFilter.js:7]: Doesn't looks like it's reverted.

**Comment by mail2bishnoi** [comment]
Something went bad. Closing this PR and opening a new one.

**Comment by mail2bishnoi** [comment]
Closing this PR and opening a newone.

**Pull request closed without merging**